mac

```diff
@@ -150,6 +150,8 @@ brew update
 
 brew_install_or_upgrade 'zsh'
 brew_install_or_upgrade 'git'
+brew_install_or_upgrade 'mongodb'
+brew_launchctl_restart 'mongodb'
```

Comment:
If we install it, we should handle the launchctl dance for it – principle of least surprise. Whether we should have it installed is another matter. Should this script include *everything* you need for all active Vermonster projects, or just the most common subset?